src/modules/scoreboard.js

```diff
@@ -0,0 +1,40 @@
+export default class Scoreboard {
+  constructor() {
+    this.scoreBoard = document.getElementById('score-board');
+    this.form = document.querySelector('form');
+    this.submitBtn = document.getElementById('submit-score');
+    this.init();
+  }
+
+  init() {
+    this.form.addEventListener('keydown', (event) => {
+      if (event.key === 'Enter') {
+        event.preventDefault();
+        this.submitBtn.click();
+      }
+    });
+    this.submitBtn.addEventListener('click', (event) => {
+      event.preventDefault();
+      this.addScore();
+    });
+  }
+
+  addScore() {
+    const userName = this.form.elements.name.value;
```

Comment:
- You did a great job by implementing the functionality for adding items however it must be just a simple structure built in HTML. you will work on the implementation of functionality in the next project, At this stage, no API calls are required. :+1: 


:+1:

Reply:
Dear @MaganaAsiati ,

Thank you for your review. However I need some clarifications, am I supposed not to have any Javascript in the project ie just html and css?